src/gen.py

```diff
@@ -20,36 +20,33 @@ def generate(target_files: List[str], mod_files: List[str]):
 def parse_options():
     # pylint: disable=W0603
     global INPUT_DIR  # pylint: disable=W0602
-    usage = f"Usage: python {__file__} [-i <input_dir>] [-o <output_dir>] " \
-            "[-T <target>] [-t <target_version>] [-d] [-f <style_format>] " \
-            "[-m <mod_version>]"
-    parser = argparse.ArgumentParser(usage)
-    parser.add_argument(
-        "-i", dest="input_dir", type=str, help="Input directory"
-    )
-    parser.add_argument(
-        "-o", dest="output_dir", type=str, help="Output directory"
+    usage = (
+        f"Usage: python {__file__} [-i <input_dir>] [-o <output_dir>] "
+        "[-T <target>] [-t <target_version>] [-d] [-f <style_format>] "
+        "[-m <mod_version>]"
     )
+    parser = argparse.ArgumentParser(usage)
+    parser.add_argument("-i", dest="input_dir", type=str, help="Input directory")
+    parser.add_argument("-o", dest="output_dir", type=str, help="Output directory")
     parser.add_argument(
-        "-f", dest="style_format", type=str,
-        help="Style format (none, yapf, ruff)"
+        "-f", dest="style_format", type=str, help="Style format (none, yapf, ruff)"
     )
     parser.add_argument(
-        "-m", dest="mod_version", type=str,
+        "-m",
+        dest="mod_version",
+        type=str,
         help="Blender version for specific mod patches to be applied "
-             "(ex. 2.79, 2.80)"
-    )
-    parser.add_argument(
-        "-T", dest="target", type=str,
-        help="Target (blender, upbge)"
+        "(ex. 2.79, 2.80)",
```

Comment:
What is the vertical alignment based format criteria between L32 an these?

Reply:
The parameters do not fit on one line, so Ruff format to have only one parameter per line.

Reply:
Thanks. I understood.